spec/requests/graphql/query/digital_object_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 require 'rails_helper'
 
-RSpec.describe 'Retrieving Field Set', type: :request do
-  let(:authorized_object) { FactoryBot.create(:item, :with_project) }
+RSpec.describe 'Retrieving Digital Object', type: :request do
```

Comment:
Ha, I resisted the urge to fix this in my branch - thanks for fixing it here.

Reply:
👍